sql/_13_issues/_23_1h/cases/cbrd_24906.sql

```diff
@@ -0,0 +1,341 @@
+drop table if exists tbl_a, tbl_b, tbl_c;
+create table tbl_a (col_a int primary key, col_b int, col_c int);
+create table tbl_b (col_a int primary key, col_b int, col_c int);
+create table tbl_c (col_a int primary key, col_b int, col_c int);
+insert into tbl_a select rownum, rownum, rownum from db_class a, db_class b, db_class c limit 1000;
+insert into tbl_b select rownum, rownum, rownum from db_class a, db_class b, db_class c limit 1000;
+insert into tbl_c select rownum, rownum, rownum from db_class a, db_class b, db_class c limit 1000;
+
```

Comment:
I think it is necessary to update statistical information.

Reply:
review reflected.